### ISSUE TITLE
[SF][ui5-input]: Clear icon should be decorative

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#10100]
### Bug Description

Since the clear icon in `ui5-input` is not accessible to keyboard users, it should be set as `decorative` mode so that the icon would be invisible to screen reader.

### Affected Component

ui5-input

### Expected Behaviour

Set the mode of icon as `Decorative`.

https://github.com/SAP/ui5-webcomponents/blob/7a23598d2afc3f7581340b766185d994e5477099/packages/main/src/Input.hbs#L49

### Steps to Reproduce

1.
2.
3.
...

### Priority

Medium

### UI5 Web Components Version

2.3.0

### Browser

Edge

### Organization

SF

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by dimovpetar** [comment]
Hello @ui5-webcomponents-topic-rl,

Please check out if this accessibility concern is valid.
In OpenUI5, the clear icon behaves the same way,

Best regards,
Petar

**Comment by ndeshev** [comment]
Hello @zhangliangyi I will further discuss this proposal with our accessibility experts, because the clear icon can be clicked, so it has some interactivity and it is not purely decorative.

**Comment by ndeshev** [comment]
Hello again, I've talked with our accessibility experts and there are ongoing discussions on the topic of this issue. We have a backlog item related to it but there is no final conclusion yet. There is also a related GitHub issue on the matter - #9870 which you can follow for solution of the topic.
